doc/nrf/ug_zigbee.rst

```diff
@@ -0,0 +1,137 @@
+The stack comes with the following features:
+
+* Complete implementation of the Zigbee core specification and Zigbee Pro feature set.
+* Support for all device roles: coordinator, router, and end device.
+* Zigbee Cluster Library (a specification for end devices).
+* Base Device Behavior.
+* Devices, described in former Zigbee Home Automation and Light Link profiles.
+* Zigbee Green Power.
+* Experimental support for ``ZB_ZCL_WWAH``.
+
+Required libraries and drivers
+******************************
+
+|zboss_lib|
+This library is available in `nrfxlib`_.
+
+The OSIF subsystem acts as the linking layer between the ZBOSS stack and |NCS|.
+It implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
+
+Configuration
+*************
+
+To use the Zigbee protocol, set the :option:`CONFIG_ZIGBEE` Kconfig option.
+Setting this option enables all the peripherals required for the correct operation of the Zigbee protocol and allows you to use them.
+
+After that, you have to define the Zigbee device role for the Zigbee application or sample by setting one of the following Kconfig options:
+
+* Router role: :option:`CONFIG_ZIGBEE_ROLE_ROUTER`
+* End device role: :option:`CONFIG_ZIGBEE_ROLE_END_DEVICE`
+* Coordinator role: :option:`CONFIG_ZIGBEE_ROLE_COORDINATOR`
+
+Setting any of these options enables the respective ZBOSS role library.
+This is needed because end devices use different libraries than routers and coordinators.
+
+Additional options
+==================
+
+After enabling the Zigbee protocol and defining the Zigbee device role, you can enable additional options in Kconfig and modify `Stack configuration options`_.
+
+You can enable the following additional configuration options:
+
+* One of the following alternative options for selecting the channel on which the Zigbee device can operate:
+
+    * :option:`CONFIG_ZIGBEE_CHANNEL_SELECTION_MODE_SINGLE` - Single mode is enabled by default.
+      The default channel is set to 16.
+      To set a different channel, edit the :option:`CONFIG_ZIGBEE_CHANNEL` option to the desired value.
+    * :option:`CONFIG_ZIGBEE_CHANNEL_SELECTION_MODE_MULTI` - In this mode, you get all the channels enabled by default.
+      To configure a custom set of channels in the range from 11 to 26, edit the :option:`CONFIG_ZIGBEE_CHANNEL_MASK` option.
+      For example, you can set channels 13, 16, and 21.
+      You must have at least one channel enabled with this option.
+* :option:`CONFIG_ZIGBEE_VENDOR_OUI` - Represents MAC Address Block Large, and by default it is set to Nordic Semiconductor's MA-L block (f4-ce-36).
+* :option:`CONFIG_ZIGBEE_CLI_LOG_ENABLED` - Enables logging of the incoming ZCL frames, and it is enabled by default.
+
+Stack configuration options
+---------------------------
+
+Zigbee is initialized after Zephyr's kernel start.
+The ZBOSS stack works on a separate Zephyr thread that is created and started with ``zigbee_enable()``.
```

Comment:
We should link to the function - but that is only possible if the API documentation is included.